### PR TITLE
Add swagger specs for groupsMembershipHistory

### PR DESCRIPTION
**Pull request by zenovich**

### CODE REVIEW

File: app/api/currentuser/get_group_memberships_history.go [hunk 1]
```diff
@@ -8,6 +8,51 @@ import (
 	"github.com/France-ioi/AlgoreaBackend/app/service"
 )
 
+// swagger:operation GET /current-user/group-memberships-history groups users groupsMembershipHistory
+// ---
+// summary: Display a history of invitations/requests for the current user
```
Comment (smadbe) [inline app/api/currentuser/get_group_memberships_history.go]: Display -> return

File: app/doc/models.go [hunk 2]
```diff
@@ -15,3 +17,26 @@ type userCreateTmpResponse struct {
...
+	// `groups_groups.sStatusDate`
+	// required: true
+	StatusDate time.Time `json:"status_date"`
+	// `groups_groups.sType`
+	// required: true
+	// enum: invitationSent,requestSent,invitationAccepted,requestAccepted,invitationRefused,requestRefused,removed,left
+	Type string `json:"type"`
+
+	// required: true
+	Group struct {
```
Comment (smadbe) [inline app/doc/models.go]: It does not seems to be really reusage in other context (as there are only name and type)...
Actually, where is it used?

File: app/api/currentuser/get_group_memberships_history.go (hunk 1 above)
Comment (smadbe) [reply app/api/currentuser/get_group_memberships_history.go]: Or just "Get a history ..." as display is already displayed at the next line.

Comment (zenovich) [reply app/api/currentuser/get_group_memberships_history.go]: fixed

File: app/doc/models.go (hunk 2 above)
Comment (zenovich) [reply app/doc/models.go]: It is used only here (by the groupsMembershipHistory service)

**Pull request merged**